appveyor.yml

```diff
@@ -44,7 +44,7 @@ test_script:
   - "%PYTHON%/Scripts/tox -e %TOX_ENV%"
 
 after_test:
-  - "%PYTHON%/python setup.py bdist_wheel"
+  - "%PYTHON%/python setup.py bdist"
```

Comment:
Hm, I think there is no point in distributing a binary if it's not wheel.
Do you know why it's failing to build a wheel in appveyor, even the wheel format package installs fine?